extension/webviews/state/store.ts

```diff
@@ -23,3 +23,9 @@ export const activeTextEditor = createState<TextEditorType>(WebviewStateType.ACT
 export const activeView = createState<ActiveView>(WebviewStateType.ACTIVE_VIEW, ActiveView.AllThreads);
 
 export const reload = createState<number>(null, 0);
+
+export const copilotMessages = createState<CopilotMessage[]>(WebviewStateType.COPILOT_MESSAGES, []);
```

Comment:
so all the copilot messages are stored in this list. It doesn't store anything in the db. that means user will lose the messages on vsc restart, i think it makes sense coz we don't wanna store these long.

Reply:
vsc restart do not erase from WebviewState